Documentation of AAD group behavior in SQL DB (#8916)
MicrosoftDocs/sql-docs: azure-sql/database/authentication-aad-overview.md

  - `SUSER_ID(<name>)`
  - `SUSER_SID(<name>)`

> - Azure SQL Database doesn't create implicit users for users logged in as part of an Azure Active Directory (Azure AD) group membership. Because of this, various operations that require assigning ownership will fail, even if the Azure AD group is added as a member to a role with those permissions.

Author: > Azure Active Directory (Azure AD)

@rwestMSFT Can we simplify this to just Azure AD?

Reviewer: @nofield I didn't check the rest of the article to see if we'd referenced it as such, but sure. It looks like we do that earlier on.

> - Azure SQL Database doesn't create implicit users for users logged in as part of an Azure Active Directory (Azure AD) group membership. Because of this, various operations that require assigning ownership will fail, even if the Azure AD group is added as a member to a role with those permissions.

Reviewer: ```suggestion
- Azure SQL Database doesn't create implicit users for users logged in as part of an Azure AD group membership. Because of this, various operations that require assigning ownership will fail, even if the Azure AD group is added as a member to a role with those permissions.
```

> For example, a user signed into a database via an Azure AD group with the **db_ddladmin** role, will not be able to execute CREATE SCHEMA, ALTER SCHEMA, and other object creation statements without a schema explicitly defined (such as table, view, or type, for example). To resolve this, an Azure AD user must be created for that user, or the Azure AD group must be altered to assign a DEFAULT_SCHEMA.

Author: @rwestMSFT Is this formatting going to place the example/resolution below the bullet point as a new paragraph? They were intended to be together in the same bullet.

> For example, a user signed into a database via an Azure AD group with the **db_ddladmin** role, will not be able to execute CREATE SCHEMA, ALTER SCHEMA, and other object creation statements without a schema explicitly defined (such as table, view, or type, for example). To resolve this, an Azure AD user must be created for that user, or the Azure AD group must be altered to assign a DEFAULT_SCHEMA.

Reviewer 2: I think we want to end this with "or the Azure AD group must be altered to assign the DEFAULT_SCHEMA to DBO"

> For example, a user signed into a database via an Azure AD group with the **db_ddladmin** role, will not be able to execute CREATE SCHEMA, ALTER SCHEMA, and other object creation statements without a schema explicitly defined (such as table, view, or type, for example). To resolve this, an Azure AD user must be created for that user, or the Azure AD group must be altered to assign a DEFAULT_SCHEMA.

Reviewer: ```suggestion
   For example, a user signed into a database via an Azure AD group with the **db_ddladmin** role, will not be able to execute CREATE SCHEMA, ALTER SCHEMA, and other object creation statements without a schema explicitly defined (such as table, view, or type, for example). To resolve this, an Azure AD user must be created for that user, or the Azure AD group must be altered to assign a DEFAULT_SCHEMA.
```

  - `SUSER_ID(<name>)`
  - `SUSER_SID(<name>)`

> - Azure SQL DB does not create implicit users for users logged in as part of an AAD group membership. Because of this various operations that require assigning ownership will fail, even if the AAD group is added as a member to a role with those permissions. For example, a user logged into a database via an AAD group with the db_ddladmin role will not be able to execute CREATE SCHEMA, ALTER SCHEMA, and other object creation statements without a schema explicitly defined (table, view, type, etc.). To resolve this, an Azure AD user must be created for that user, or the AAD group must be altered to assign a DEFAULT_SCHEMA.

Reviewer: ```suggestion
- Azure SQL Database doesn't create implicit users for users logged in as part of an Azure Active Directory (Azure AD) group membership. Because of this, various operations that require assigning ownership will fail, even if the Azure AD group is added as a member to a role with those permissions.

For example, a user signed into a database via an Azure AD group with the **db_ddladmin** role, will not be able to execute CREATE SCHEMA, ALTER SCHEMA, and other object creation statements without a schema explicitly defined (such as table, view, or type, for example). To resolve this, an Azure AD user must be created for that user, or the Azure AD group must be altered to assign a DEFAULT_SCHEMA.
```
